client/src/components/layout/ReviewTile.js

```diff
@@ -1,7 +1,21 @@
-import React, { useState } from "react"
+import React, { useEffect, useReducer, useState } from "react"
 import ReviewEditButton from "./ReviewEditButton"
 import ReviewDeleteButton from "./ReviewDeleteButton"
 const ReviewTile = ({ review, user, setPlanet, planet }) => {
+  const [currentUserVoteValue, setCurrentUserVoteValue] = useState(0)
```

Comment:
See below. Once you update to have this valued computed in a serializer, you wont need this state. Delete!